src/adapter/src/catalog/builtin.rs

```diff
@@ -2164,6 +2164,33 @@ UNION ALL
     SELECT id, oid, schema_id, name, 'secret', owner_id, privileges FROM mz_catalog.mz_secrets",
 };
 
+pub const MZ_OBJECT_FULLY_QUALIFIED_NAMES: BuiltinView = BuiltinView {
+    name: "mz_object_fully_qualified_names",
+    schema: MZ_CATALOG_SCHEMA,
+    sql: "CREATE VIEW mz_catalog.mz_object_fully_qualified_names (id, name, schema_name, database_name) AS
+    SELECT o.id, o.name, sc.name as schema_name, db.name as database_name
```

Comment:
Let's add object type to this view, so that I can easily join this with the lifetimes view.

Reply:
Done!